frontend/stories/components/ContentDisplayToggle.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import { Meta } from "@storybook/react";
+import { identity } from "lodash";
+
+import ContentDisplayToggle from "src/components/ContentDisplayToggle";
+
+const meta: Meta<typeof ContentDisplayToggle> = {
+  title: "Components/ContentDisplayToggle",
+  component: ContentDisplayToggle,
+};
+export default meta;
+
+export const Default = {
```

Comment:
this is not fully interactive, would require adding some add ons. Do we want to explore that? https://storybook.js.org/docs/addons/addons-api#useargs

Reply:
I wonder if we should make a followup ticket to document this idea, so we can discuss in eng sync and possibly explore it in the future?

Reply:
https://github.com/HHS/simpler-grants-gov/issues/2331